### PR TITLE
zh-CN: create Glossary/Speed_index

### PR DESCRIPTION
**Pull request by PassionPenguin**

### CODE REVIEW

File: files/zh-cn/glossary/speed_index/index.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+title: 速度索引
+slug: Glossary/Speed_index
+l10n:
+  sourceCommit: ada5fa5ef15eadd44b549ecf906423b4a2092f34
+---
+
+{{GlossarySidebar}}
+
+**速度索引**（SI）是一个页面加载性能指标，用于衡量页面内容的可见加载速度。速度索引取决于视口的大小，以毫秒为单位表示：时间越短，得分越高。
```
Comment (yin1999) [inline files/zh-cn/glossary/speed_index/index.md:10]:
```suggestion
**速度指数**（SI）页面加载性能指标用于衡量页面可见内容的加载速度。速度索引取决于视口的大小，以毫秒为单位表示：时间越短，得分越高。
```

File: files/zh-cn/glossary/speed_index/index.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+速度索引是为了解决其他里程碑和指标的问题，提供真实用户体验的度量。速度索引已经在几个常见的审核中实现，包括[WebPageTest](https://github.com/WPO-Foundation/webpagetest-docs/blob/main/src/metrics/SpeedIndex.md) 和 [Lighthouse](https://github.com/paulirish/speedline)。
```
Comment (yin1999) [inline files/zh-cn/glossary/speed_index/index.md:12]:
```suggestion
速度指数是为了解决其他里程碑和指标的问题，提供真实用户体验的度量。速度指数已经在几个常见的审核中实现，包括 [WebPageTest](https://github.com/WPO-Foundation/webpagetest-docs/blob/main/src/metrics/SpeedIndex.md) 和 [Lighthouse](https://github.com/paulirish/speedline)。
```

File: files/zh-cn/glossary/speed_index/index.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+速度索引的计算方式是：在页面视觉上完全加载之前的每 100 毫秒间隔内，页面的视觉完成百分比。总分是未视觉完成的屏幕百分比的 10 次每秒间隔的总和。
```
Comment (yin1999) [inline files/zh-cn/glossary/speed_index/index.md:14]: 最后一句话的意思我不是太能理解

File: files/zh-cn/glossary/speed_index/index.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+_显示了页面加载事件之前的视窗内容如何加载，并由速度索引进行测量的折线图_：
```
Comment (yin1999) [inline files/zh-cn/glossary/speed_index/index.md:16]:
```suggestion
_显示了页面加载事件之前的首屏内容如何加载和由速度指数测量所得的图表_：
```

File: files/zh-cn/glossary/speed_index/index.md (hunk 3 above)
Comment (PassionPenguin) [reply files/zh-cn/glossary/speed_index/index.md:14]: Speed Index is calculated by what percent of the page is visually complete at every 100ms interval until the page is visually complete. The overall score is a sum of the individual 10 times per second intervals（一样是 100ms 的间隔了） of the percent of the screen that is not visually complete.

```suggestion
速度指数的计算方式是：在每 100 毫秒间隔内直至页面在视觉上完全加载，页面在视觉上加载完成的百分比。指数的总分则是同样间隔内未完全显示的屏幕百分比之和。
```

File: files/zh-cn/glossary/speed_index/index.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+  sourceCommit: ada5fa5ef15eadd44b549ecf906423b4a2092f34
+---
+
+{{GlossarySidebar}}
+
+**速度指数**（SI）页面加载性能指标用于衡量页面可见内容的加载速度。速度指数取决于视口的大小，以毫秒为单位表示：时间越短，得分越高。
+
+速度指数是为了解决其他里程碑和指标的问题，提供真实用户体验的度量。速度指数已经在几个常见的审核中实现，包括 [WebPageTest](https://github.com/WPO-Foundation/webpagetest-docs/blob/main/src/metrics/SpeedIndex.md) 和 [Lighthouse](https://github.com/paulirish/speedline)。
+
+速度指数的计算方式是：在每 100 毫秒间隔内直至页面在视觉上完全加载，页面在视觉上加载完成的百分比。指数的总分则是同样间隔内未完全显示的屏幕百分比之和。
```
Comment (yin1999) [inline files/zh-cn/glossary/speed_index/index.md:14]:
```suggestion
速度指数的计算方式是：在页面可视内容完全加载之前，每隔 100 毫秒计算一次页面可视内容加载完成的百分比。指数的总分则是同样间隔内未完全显示的屏幕百分比之和。
```

**Review by yin1999 — COMMENTED** [review]
前一句我感觉还是按最初版本的翻译改一下比较好

**Review by yin1999 — APPROVED** [review]

**Pull request merged**